Trustroots/trustroots
# Migrate Tribe to React

#### Proposed Changes

Make tribe into a react component.

![image](https://user-images.githubusercontent.com/7449720/69738321-339c2a00-1136-11ea-9cf2-24337c78fd51.png)

#### Testing Instructions

* See that design and functionality survived the migration
* See that the tests and lint pass

Part of #1133 
~Depends on #1123, should be reviewed after that one is merged to the feature branch `tribes-react`~
